package.json

```diff
@@ -105,16 +103,15 @@
         "ophan-tracker-js": "^1.3.23",
         "pm2": "^4.5.0",
         "polished": "^1.9.2",
-        "preact": "^10.3.1",
-        "preact-render-to-string": "^5.1.10",
+        "preact": "^10.5.7",
```

Comment:
Makes [this Snyk nag](https://github.com/guardian/dotcom-rendering/pull/2113) moot